src/datascience-ui/data-explorer/mainPanel.tsx

```diff
@@ -154,33 +156,52 @@ export class MainPanel extends React.Component<IMainPanelProps, IMainPanelState>
                 />
                 {progressBar}
                 {this.renderBreadcrumb()}
+                {this.renderSliceControls()}
                 {this.state.totalRowCount > 0 && this.state.styleReady && this.renderGrid()}
             </div>
         );
     };
 
-    private renderBreadcrumb() {
-        if (this.state.fileName) {
-            let breadcrumbText = this.state.variableName;
-            if (this.state.originalVariableShape) {
-                breadcrumbText += ' (' + this.state.originalVariableShape?.join(', ') + ')';
-            }
+    public renderSliceControls = () => {
```

Comment:
Moved rendering SliceControl out of ReactSlickGrid, there really wasn't a reason for it to be a child of the grid